Sources/SwiftDocCUtilities/ArgumentParsing/Subcommands/Convert.swift

```diff
@@ -528,7 +528,12 @@ extension Docc {
                 """)
             )
             var enableExperimentalLinkHierarchySerialization = false
-            
+
+            @Flag(
+                name: .customLong("enable-experimental-mentioned-in"),
+                help: ArgumentHelp("Enable automatic generation of links on symbols to articles that mention that symbol"))
```

Comment:
minor: There's something about the "links on symbols to articles" phrasing that I find hard to read.

IMO replacing "symbols" with "symbol pages" helps clarify which is the page and which is the link"

> Enable automatic generation of links on symbol pages to articles that mention that symbol

```suggestion
                help: ArgumentHelp("Enable automatic generation of links on symbol pages to articles that mention that symbol"))
```

Still, I find that the distance between "links" and "mention" makes me go back and re-read the sentence to connect that the links are to the mentioning articles. Moving up "mention" before "link" helps me understand that the generated links are about mentions before getting to the end of the sentence

> Enable automatic generation of "mentioned in" links on symbol pages to articles that mention that symbol

That adds an unnecessary repetition of "mention". The latter mention could be replaced with "links" but that introduces a repetition of "links", once as a noun and once as a verb, which might be more confusing. As an alternative, the noun "links" could be replaced with "section" or the verb "links" could be replaced with "reference"

> Enable automatic generation of "mentioned in" sections on symbol pages that list articles that links to that symbol

> Enable automatic generation of "mentioned in" links on symbol pages to articles that reference that symbol

or both

> Enable automatic generation of "mentioned in" sections on symbol pages that list articles which references that symbol

In either case, I feel that the word "automatic" is redundant here.

```suggestion
                help: ArgumentHelp("Enable generation of "mentioned in" sections on symbol pages that list articles which references that symbol"))
```

If it's not important to convey that the mentioned in sections only list articles, then the sentence could be shortened to 

> Enable generation of "mentioned in" sections on symbol pages


```suggestion
                help: ArgumentHelp("Enable generation of "mentioned in" sections on symbol pages"))
```

Reply:
I'll update this.